src/feature/svgLegend/index.js

```
@@ -0,0 +1,371 @@
+  let iconY;
+  const g = document.createElementNS(svgNS, 'g');
+  g.setAttribute('style', `--inactiveColor:${inactiveColor};`);
+  g.setAttribute('class', `legend-svg-item`);
+  g.setAttribute('index', index);
+  switch (type) {
+    case 'circle':
+      const radius = (width > height ? height : width) / 2;
+      icon = document.createElementNS(svgNS, 'circle');
+      icon.setAttribute('cx', (x + itemGap * index) + radius*2);
+      icon.setAttribute('cy', y + radius + 2);
+      icon.setAttribute('r', radius);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)